nautobot/dcim/migrations/0034_remove_region_and_site.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.17 on 2023-02-21 21:47
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
Can we leave a comment here reminding ourselves when we re-order this file to ensure the docs are updated for the migration guide?